docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -1,84 +1,160 @@
+
+    <br>
+
+    <tabset>
+        <tab heading="CLI">
+            <br>
+            <ul>
+                <li>If using the CLI, the UX Aspects stylesheet can be added in the <code>angular-cli.json</code> file:
+                    <br><br>
+                    <uxd-snippet [code]="cliJson" language="javascript"></uxd-snippet>
+                </li>
+            </ul>
+        </tab>
+        <tab heading="Regular Webpack">
+            <br>
+            <ul>
+                <li>When using regular webpack, UX Aspects can be included in a stylesheet by using:
+                    <br><br>
+                    <uxd-snippet [code]="stylesheetCss" language="css"></uxd-snippet>
+                </li>
+            </ul>
+        </tab>
+        <tab heading="JavaScript/Typescript">
+            <br>
+            <ul>
+                <li>The UX Aspects stylesheet can also be included directly from JavaScript/TypeScript by using webpack:
+                    <br><br>
+                    <uxd-snippet [code]="webpackJs" language="javascript"></uxd-snippet>
+                </li>
+            </ul>
+        </tab>
+        <tab heading="Non-Angular Sites">
+            <br>
+            <ul>
+                <li>The UX Aspects stylesheet can be applied by adding it to the header of the site as follows:
+                    <br><br>
+                    <uxd-snippet [code]="headerHtml" language="html"></uxd-snippet>
+                    <p>Please note this is the only required step for non-Angular sites. However, this method is not recommended with
+                        the use of <code>node_modules</code>.</p>
+                </li>
+            </ul>
+        </tab>
+    </tabset>
+
+    <br>
 
     <h3>Using UX Aspects</h3>
+    
+    <div class="row-divider"></div>
 
-    <p>Add the UX Aspects stylesheet to the header of the site. This is the only required step for non-Angular sites.</p>
+    <p>UX Aspects can be used in Angular 4 and Angular JS projects as follows:</p>
 
-    <uxd-snippet [code]="stylesheetHtml" language="html"></uxd-snippet>
+    <br>
 
-    <h4>In an Angular project</h4>
+    <tabset>
+        <tab heading="Angular 4">
+            <br>
 
-    <p>Configure TypeScript to search for the UX Aspects module in the correct location by adding the following to your <code>tsconfig.json</code> file:</p>
+            <p>Import the required component module(s) into the module that will be using them.</p>
 
-    <uxd-snippet language="javascript" [code]="tsconfig"></uxd-snippet>
+            <uxd-snippet [code]="angular4Ts" language="javascript"></uxd-snippet>
 
-    <p>Import the required component(s) into the module that will be using them.</p>
+            <p>Please note if using Bower, you must specify the relative path to the bower package, e.g. 
+                <code>import {{ '{' }} CheckboxModule {{ '}' }} from '../../bower_components/ux-aspects';</code></p>
```

Comment:
Use a snippet if this must be cited here. (I'm not convinced that it's necessary, and it's a conditional step which is what we are trying to avoid with these tabsets.)